index.html

```diff
@@ -160,10 +148,25 @@ <h1>The OpenTF Manifesto</h1>
         </ul>
 
         <p class="font-bold">
-          We ask HashiCorp to do right by the community and donate Terraform to
-          a reputable foundation where, together with the community and
-          resources that we pledge, we continue to build and maintain this
-          amazing ecosystem.
+          Our request of HashiCorp: donate Terraform to this foundation.
+        </p> 
+        
+        <p>
+          We ask HashiCorp to do the right thing by the community and to donate Terraform to this foundation,
+          rather than going forward with the BSL license change. That way, instead of fracturing the community, 
+          we end up with a single, impartial, reliable home for Terraform where the whole community can come together 
+          to keep building this amazing ecosystem.
+        </p>        
+
+        <p class="font-bold">
+          Our fallback plan: fork Terraform.
+        </p>
+
+        <p>
+          If HashiCorp is unwilling to donate Terraform to the foundation, then we propose to fork Terraform, and 
```

Comment:
I don't think we should have an ultimatum

Reply:
+1

Reply:
I don’t think it should be aggressively phrased as an ultimatum but needs to be clear to the reader that this is not a whimpering request.